ci/after-build-check-tasks.js

```diff
@@ -105,33 +126,74 @@ function findNonUniqueTaskLib() {
         });
     }
 
-    const haveDependencies = findWithFsFromPaths(scanningTasks);
+    return scanningTasks;
+}
+
+/**
+ * Function looking for multiple azure-pipelines-task-lib versions
+ * in builded tasks, in case if package found multiple times throw error
+ * Note: now function compares only for tasks which have Node10 and Node16 in their task.json
+ */
+function findNonUniqueTaskLib() {
+    const taskLibSection = "#findnonuniquetasklib-section"
+    const scanningTasks = getBuiltTasks();
+    const haveDependencies = findTaskLibWithFsFromPaths(scanningTasks);
     if (haveDependencies.length > 0) {
-        console.log(`##vso[task.logissue type=error;sourcepath=ci/check-tasks.js;linenumber=109;]The following tasks have duplicate azure-pipelines-task-lib: 
-            ${JSON.stringify(haveDependencies, null, 2)}
-            Please examine the following link: ${GITHUB_LINK}`);
+        logToPipeline('error', `The following tasks have duplicate azure-pipelines-task-lib:\n${JSON.stringify(haveDependencies, null, 2)}`);
+    logToPipeline('error', `Please examine the following link: ${GITHUB_LINK + taskLibSection}`);
         process.exit(1);
     }
 
-    console.log('No duplicates found.');
+    logToPipeline('info', 'No duplicates found');
     return null;
 }
 
 function analyzePowershellTasks() {
     let output = '';
     if (process.platform !== 'win32') {
-        console.log('The powershell check is only supported on Windows. Skipping...');
+        logToPipeline('info', 'The powershell check is only supported on Windows. Skipping...');
         return;
     }
 
     try {
         const pwshScriptPath = path.join(__dirname, 'check-powershell-syntax.ps1');
         output = util.run(`powershell -NoLogo -Sta -NoProfile -NonInteractive -ExecutionPolicy Unrestricted ${pwshScriptPath} ${buildTasksPath}`, true);
     } catch (e) {
-        console.log(`##vso[task.logissue type=error;sourcepath=ci/check-tasks.js;linenumber=123;]Please check the tasks, seems like they have invalid PowerShell syntax.`)
+        logToPipeline('error', 'Please check the tasks, seems like they have invalid PowerShell syntax.');
         process.exit(1);
     }
 }
 
-// findNonUniqueTaskLib();
-analyzePowershellTasks();
\ No newline at end of file
+function findIncompatibleAgentBase() {
+    const minAgentBaseVersion = '6.0.2';
```

Comment:
is this version compatible with node 10-16?

Reply:
yes